pkg/controller/constants/image.go

```diff
@@ -14,20 +14,31 @@ const (
 	DefaultOperandImageOpenJ9 = "infinispan-for-openj9/server:latest"
 )
 
+var (
+	// Internal purpose variable for lazy initialization
+	defaultImage string
+)
+
 // GetDefaultInfinispanJavaImage returns default Infinispan Java image depends of the runtime architecture
 func GetDefaultInfinispanJavaImage() string {
+	// If the variable was already initialize, return it
+	if defaultImage != "" {
+		return defaultImage
+	}
+
 	// Full list of archs might be found here:
 	// https://github.com/golang/go/blob/release-branch.go1.10/src/go/build/syslist.go#L8
 	switch arch := runtime.GOARCH; arch {
 	case "ppc64le", "s390x":
-		return GetEnvWithDefault(OperandImageOpenJ9, DefaultOperandImageOpenJ9)
+		defaultImage = GetEnvWithDefault(OperandImageOpenJ9, DefaultOperandImageOpenJ9)
 	default:
 		// Try new name RELATED_IMAGE_OPENJDK first
 		retVal := GetEnvWithDefault(OperandImageOpenJDK, "")
```

Comment:
We can use following code here and don't use if-then section later
```suggestion
		retVal := GetEnvWithDefault(OperandImageOpenJDK, GetEnvWithDefault("DEFAULT_IMAGE", DefaultOperandImageOpenJDK))
```